iam/cloud-client/snippets/create_key.py

```diff
@@ -0,0 +1,61 @@
+# Copyright 2022 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+# This file contains code samples that demonstrate how to get create IAM key for service account.
+
+# [START iam_create_key]
+from google.cloud import iam_admin_v1
+from google.cloud.iam_admin_v1 import types
+
+
+def create_key(project_id: str, account: str) -> str:
+    """
+    Creates a key for a service account.
+
+    project_id: ID or number of the Google Cloud project you want to use.
+    account: ID or email which is unique identifier of the service account.
+    """
+
+    iam_admin_client = iam_admin_v1.IAMClient()
+    request = types.CreateServiceAccountKeyRequest()
+    request.name = f"projects/{project_id}/serviceAccounts/{account}"
+
+    key = iam_admin_client.create_service_account_key(request=request)
+    key_id = ""
+
+    if not key.disabled:
+        # The private_key_data field contains the stringified service account key
+        # in JSON format. You cannot download it again later.
+        import json
+        json_key_data = json.loads(key.private_key_data)
+        print(f"Created a key: {json_key_data['private_key_id']}")
+        key_id = json_key_data["private_key_id"]
+
+    return key_id
```

Comment:
Perhaps we should return `json_key_data` or even better, the whole `key`, as you warn that it cannot be obtain later - let's not lose it that fast :P

Reply:
Yeah, makes sense 😅